src/main/java/core/basesyntax/strategy/handlers/OperationHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.handlers;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    default Boolean isValidTransactionData(String fruit, int quantity) {
```

Comment:
You can throw an exception with "invalid data transaction" here to avoid code duplication in other handlers